test/real_eq_operator_test.cpp

```diff
@@ -936,8 +936,56 @@ TEST_CASE("Operator ==") {
                 boost::real::real c = a * b;
                 boost::real::real d("1.1111111111111");
                 boost::real::real e = d * b;
-                CHECK_THROWS_AS(c == e, boost::real::precision_exception);
+                CHECK_FALSE(c == e);
+            }
+        }
+    }
+
+    SECTION("My Tests") {
```

Comment:
The name can't be "My test". The name must be descriptive about what the section is testing. This is an open source code that more people will work on it and not a private code.

Reply:
Sorry... I intended that as a temporary but forgot to remove it.